tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb

```diff
@@ -1250,9 +768,263 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.18"
+   "version": "3.9.0"
+  },
+  "widgets": {
+   "application/vnd.jupyter.widget-state+json": {
+    "08b427ccdf704a2cb3229c4465abde44": {
+     "model_module": "@jupyter-widgets/output",
+     "model_module_version": "1.0.0",
+     "model_name": "OutputModel",
+     "state": {
+      "_dom_classes": [],
+      "_model_module": "@jupyter-widgets/output",
+      "_model_module_version": "1.0.0",
+      "_model_name": "OutputModel",
+      "_view_count": null,
+      "_view_module": "@jupyter-widgets/output",
+      "_view_module_version": "1.0.0",
```

Comment:
why is there widget state?

Reply:
Sorry, no idea.
I took that video rendering code from other tutorials.

Now I replaced the previous video rendering cell with the one given under the template  [video1 here](https://colab.research.google.com/github/neuromatch/course-content-template/blob/main/tutorials/W1D2_Template/student/W1D2_Tutorial1.ipynb#scrollTo=t9fk3pRD8dGM)

I hope it works fine now.